_source/logzio_collections/_log-sources/aws-security-hub.md

```diff
@@ -0,0 +1,83 @@
+---
+title: AAWS Security Hub 
+logo:
+  logofile: aws.svg
+  orientation: vertical
+data-source: AWS Security Hub
+templates: ["lambda-cloudwatch"]
+open-source:
+  - title: AWS Security Hub
+    github-repo: aws-security-hub
+contributors:
+  - nshishkin
+shipping-tags:
+  - aws
+order: 830
+---
+
+#### Configuration
+
+<!-- info-box-start:info -->
+Your Lambda function needs to run within the AWS Lambda limits, such as memory allocation and timeout. Make sure you understand these limits. If you can't adjust your settings to stay within the Lambda limits, you can use the AWS [Support Center console](https://console.aws.amazon.com/support/v1#/case/create?issueType=service-limit-increase) to request an increase. [Learn more about AWS Lambda Limits](https://docs.aws.amazon.com/lambda/latest/dg/limits.html).
+{:.info-box.important}
+<!-- info-box-end -->
+
+This deployment will automatically create the following resources:
+
+![Resources](https://dytvr9ot2sszz.cloudfront.net/logz-docs/aws/resources-security-hub.png)
+
+This integration ships events from AWS Security Hub to Logz.io.
+
+A new event triggers a designated EventBridge rule, which invokes a Lambda function. The function processes the event and sends it to Logz.io.
```

Comment:
Can we move it to the top of the page? I feel like we need to do a short description of the purpose/integration before diving into details.